artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/cursor/PageSubscription.java

```diff
@@ -161,4 +165,9 @@
     * @throws Exception
     */
    void onDeletePage(Page deletedPage) throws Exception;
+
+   AtomicLong getDeliveredCount();
```

Comment:
not sure would want to expose an atomic outside the object as it allows other code to update it unexpectedly, should probably be  primitive and the object should return get on the atomic.

Reply:
Good point, I will make that change.